pkg/metastore/ms_client.go

```diff
@@ -55,22 +59,30 @@ func CopyDB(ctx context.Context, fromClient, toClient Client, fromDB, toDB, toBr
 			return "", nil
 		}
 		location = HandleDBFSLocation(location, dbfsLocation)
-		return ReplaceBranchName(location, toBranch)
+		transformedLocation, err := ReplaceBranchName(location, toBranch)
+		if err != nil {
+			return "", fmt.Errorf("failed to replace branch name with location: %s and branch: %s : %w", location, toBranch, err)
+		}
+		return transformedLocation, nil
 	}
 	return copyDBWithTransformLocation(ctx, fromClient, toClient, fromDB, toDB, transformLocation)
 }
 
 func copyDBWithTransformLocation(ctx context.Context, fromClient, toClient Client, fromDB string, toDB string, transformLocation func(location string) (string, error)) error {
 	schema, err := fromClient.GetDatabase(ctx, fromDB)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to get database on copy: %w", err)
 	}
 	schema.Name = toDB
 	schema.LocationURI, err = transformLocation(schema.LocationURI)
 	if err != nil {
 		return err
 	}
-	return toClient.CreateDatabase(ctx, schema)
+	err = toClient.CreateDatabase(ctx, schema)
+	if err != nil {
+		return fmt.Errorf("failed to create database with name %s and location %s   :%w", schema.Name, schema.LocationURI, err)
```

Comment:
- fix the spaces
- prefer to use '%s' to quote names/values

Reply:
Done